packages/patternfly-3/patternfly-react/CONTRIBUTING.md

```diff
@@ -15,95 +15,6 @@
 4. Add tests to `[ComponentName].test.js`. All new components must be tested.
 5. Update the generated storybook `[ComponentName].stories.js`. When your stories contain multiple files, put them in a subfolder named `Stories`. Stories should demonstrate as many different UI states for your component as possible. Use Storybook knobs to enable dynamic visualizations of your component's props. For PatternFly 4 components, provide associated examples for documentation in the examples directory for the component.
 
-## Code Contribution Guidelines
```

Comment:
Where are these steps now?

Reply:
These steps are out of date and catered towards adding components (which is not done often). Most of them are how to use `git` (which isn't specific to our project), and the only thing left that's relevant is `yarn generate` for scaffolding new components, which we can document better elsewhere. `yarn test` is rather obvious, and `yarn commit` isn't required anymore since semantic commits aren't required.